packages/react/scroll-area/src/ScrollArea.tsx

```diff
@@ -747,7 +747,22 @@ interface ScrollAreaThumbProps extends PrimitiveDivProps {
 
 const ScrollAreaThumb = React.forwardRef<ScrollAreaThumbElement, ScrollAreaThumbProps>(
   (props: ScopedProps<ScrollAreaThumbProps>, forwardedRef) => {
-    const { __scopeScrollArea, forceMount, style, ...thumbProps } = props;
+    const { forceMount, __scopeScrollArea, ...contentProps } = props;
+    const scrollbarContext = useScrollbarContext(THUMB_NAME, __scopeScrollArea);
```

Comment:
```suggestion
    const { forceMount, ...thumbProps } = props;
    const scrollbarContext = useScrollbarContext(THUMB_NAME, props.__scopeScrollArea);
```

We should ensure  `__scopeScrollArea` gets spread into the `impl` part as well